R/type-bare.R

```diff
@@ -7,77 +7,49 @@
 #' @method vec_ptype2 logical
 #' @export
 vec_ptype2.logical <- function(x, y, ..., x_arg = "", y_arg = "") {
-  if (is.object(x)) {
-    vec_default_ptype2(x, y, x_arg = x_arg, y_arg = y_arg)
-  } else {
-    UseMethod("vec_ptype2.logical")
-  }
+  UseMethod("vec_ptype2.logical")
```

Comment:
Can you add a note in the vignette saying that these functions are never actually called?